src/OrchardCore.Modules/OrchardCore.Search.Elasticsearch/Drivers/ContentTypePartIndexSettingsDisplayDriver.cs

```diff
@@ -0,0 +1,53 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using OrchardCore.ContentManagement.Metadata.Models;
+using OrchardCore.ContentTypes.Editors;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.Search.Elasticsearch.Core.Models;
+using OrchardCore.Search.Elasticsearch.ViewModels;
+
+namespace OrchardCore.Search.Elasticsearch.Drivers
+{
+    public class ContentTypePartIndexSettingsDisplayDriver : ContentTypePartDefinitionDisplayDriver
```

Comment:
No, it will not work unless you set the "subtitle" field as part of the ContentItem.FullText setting. Else, you need to add the BlogPost.Subtitle field in the Elasticsearch search settings.

Reply:
First method. Adding the BlogPost.Subtitle in the Search allowed fields.

https://user-images.githubusercontent.com/3228637/186556883-58a64e8e-94a2-4853-8bf1-124585e310dd.mp4

Reply:
Also, fixed the FullText indexing as part of [9c37687](https://github.com/OrchardCMS/OrchardCore/pull/11052/commits/9c37687c6a6824f797037555929642596fb70bc9)

Reply:
Ah of course, right.